language-command.php

```diff
@@ -41,3 +29,9 @@
 if ( class_exists( 'WP_CLI\Dispatcher\CommandNamespace' ) ) {
 	WP_CLI::add_command( 'language', 'Language_Namespace' );
 }
+
+function wpcli_language_check_requirements() {
+	if ( \WP_CLI\Utils\wp_version_compare( '4.0', '<' ) ) {
+		WP_CLI::error( 'Requires WordPress 4.0 or greater.' );
+	}
+}
```

Comment:
If it's a closure, we need to terminate the statement here:
```suggestion
};
```